features/com.contrastsecurity.ide.eclipse.feature/feature.properties

```diff
@@ -1,26 +1,11 @@
-/*******************************************************************************
```

Comment:
File corresponding to features/com.contrastsecurity.ide.eclipse.feature/feature_ja.properties